addons/digest/data/digest_data.xml

```diff
@@ -12,13 +12,16 @@
         <meta http-equiv="X-UA-Compatible" content="IE=9; IE=8; IE=7; IE=EDGE" />
 
         <style type="text/css">
+            :root {
```

Comment:
Could you make a specific commit with changes in digest (with its own explanation, ...) ? It is not really linked to "templates usability".